test/Microsoft.AspNet.Mvc.FunctionalTests/TestHelper.cs

```diff
@@ -81,10 +113,11 @@ public static TestServer CreateServer(Action<IApplicationBuilder> builder, strin
                 applicationBasePath,
                 applicationWebSiteName);
             services.AddInstance<IApplicationEnvironment>(environment);
+            services.AddInstance<IHostingEnvironment>(new HostingEnvironment(environment));
```

Comment:
@HaoK please revert the changes to this file that were not required.  we don't need unused helper methods lying around.